src/state/download/reducer.js

```diff
@@ -52,14 +52,7 @@ export default (state = initialState, action) => {
         isLoading: false
       };
     }
-    case 'DOWNLOAD_REMOVE_EXPERIMENT_SUCCESS': {
-      const { dataSet } = action.data;
-      return {
-        ...state,
-        dataSet
-      };
-    }
-    case 'DOWNLOAD_REMOVE_SPECIES_SUCCESS': {
+    case 'DOWNLOAD_UPDATE_DATASET': {
```

Comment:
Just curious: why some actions has a "xxx_SUCCESS" affiliated but some don't? Thanks.

Reply:
I think the main reason is when we need to track the state of a request in the redux store. For example, if we needed a resource named `FOO`, we could have the actions `FETCH_FOO` and `FETCH_FOO_SUCCESS`. Here we were associating two actions for each request that we needed to do to the server.

In this specific case, I guess this action could also have been called `DOWNLOAD_UPDATE_DATASET_SUCCESS`, since it's called mostly after a request to the server that modifies the dataset. However I saw it as something more general, we can use it every time that we want to update the dataset in the store.

This is not the only way to do it, in some cases it's simpler to track the state of the requests on the views. For that we have the component [Loader](https://github.com/AlexsLemonade/refinebio-frontend/blob/dev/src/components/Loader/index.js). I wanted to start doing refactors to gradually simplify the store and the action creators, and probably remove some of these actions.

Do you think this makes sense??

Reply:
Thanks @arielsvn. It makes sense. :+1: